csi/node.go

```diff
@@ -389,3 +412,36 @@ func ensureMountPathCreated(targetPath string) error {
 
 	return nil
 }
+
+func makeFile(pathname string) error {
+	f, err := os.OpenFile(pathname, os.O_CREATE, os.FileMode(0644))
+	defer func() {
+		err := f.Close()
+		if err != nil {
+			logrus.Warnf("failed to close file: %s", err.Error())
+		}
+	}()
+	if err != nil {
+		if !os.IsExist(err) {
+			return err
+		}
+	}
+	_, err = f.Stat()
```

Comment:
Just curious on this style. Don't want to check any stat values? May be just add some comments to why this style is being used.

Reply:
Removed the stat call, it was leftover from the original implementation.